src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -6,1136 +6,200 @@ namespace Microsoft.Azure.ServiceBus
     using System;
     using System.Collections.Generic;
     using System.Globalization;
-    using System.IO;
-    using System.Runtime.Serialization;
-    using System.Threading;
-    using Core;
-    using Primitives;
+    using System.Text;
 
-    /// <summary>Represents the unit of communication between ServiceBus client and Service.</summary>
-    public sealed class Message : IDisposable
+    public class Message
```

Comment:
per @vinaysurya this should still implement IDisposible. How about being sealed?